### PR TITLE
[4.0] [a11y] atum Underlined links

### PR DESCRIPTION
**Pull request by himanshu007-creator**
Pull Request for Issue #33629 .

### Summary of Changes
The links in list earlier were not underlined.

### Testing Instructions
Apply this patch , compile css and have a look on links in dashboard 

### Actual result BEFORE applying this Pull Request
Links were not underlined

### Expected result AFTER applying this Pull Request
Links are underlined

p2

### CODE REVIEW

**Comment by brianteeman** [comment]
Sorry but this is completely wrong. You should be using css here.

**Comment by richard67** [comment]
@himanshu007-creator Sorry, but this PR is wrong. We don't use anywhere the `<u>` element to underline links, we use CSS for that.

Furthermore, as you can read here, it shall not be used for underlining links: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u

> Be careful to avoid using the `<u>` element with its default styling (of underlined text) in such a way as to be confused with a hyperlink, which is also underlined by default.

**Comment by richard67** [comment]
@brianteeman Seems you were a few seconds faster than me ;-)

**Comment by himanshu007-creator** [comment]
@richard67  @brianteeman  thanks for speedy feedback, really appreciate that!. I get your point. I will try to make changes using CSS only. I will surely work on it tomorrow :)

**Comment by himanshu007-creator** [comment]
I have used BS5 <code>text-decoration-underline</code> for this. <a href="https://getbootstrap.com/docs/5.0/utilities/text/#text-decoration"><i>reference</i></a> 
is it good to go, or do i have to make changes in CSS itself?

**Comment by brianteeman** [comment]
@himanshu007-creator this is how I would approach this problem.

1. Look at the other underlined links and see how they have been styled
2. Compare the two and then work to apply the **same** styling

**Comment by richard67** [comment]
My gut feeling is that the styling is already somewhere, and it just needs to modify the CSS selector for it.

**Comment by himanshu007-creator** [comment]
@richard67  i am thinking of this 
<code>   a[class^="flex-grow-1"] {
      text-decoration: underline;
    }
</code>

in file administrator/templates/atum/scss/pages/_com_cpanel.scss

File: administrator/templates/atum/scss/pages/_com_cpanel.scss [hunk 1]
```diff
@@ -174,6 +174,10 @@
     &:hover {
       background: var(--toolbar-bg);
     }
+    
+   a[class^="flex-grow-1"] {
+     text-decoration: underline;
```
Comment (richard67) [inline administrator/templates/atum/scss/pages/_com_cpanel.scss]:
```suggestion
    a[class^="flex-grow-1"] {
      text-decoration: underline;
```
Please indent in the same way as you can see in previous code parts. Code style is important for quick readability and so maintainability of source code.

**Comment by Quy** [comment]
Closing in favor of PR #33808. Thanks.

**Pull request closed without merging**